### ISSUE TITLE
Error running Telescope neoclip unnamed

### ISSUE THREAD

**Issue by desdic** [AckslD/nvim-neoclip.lua#48]
When running `:Telescope neoclip unnamed` I get this error
```
E5108: Error executing lua ...scope.nvim/lua/telescope/finders/async_static_finder.lua:16: bad argument #1 to 'ipairs' (table expected, got nil)
stack traceback:
        [C]: in function 'ipairs'
        ...scope.nvim/lua/telescope/finders/async_static_finder.lua:16: in function 'new_table'
        .../packer/start/nvim-neoclip.lua/lua/neoclip/telescope.lua:158: in function <.../packer/start/nvim-neoclip.lua/lua/neoclip/telescope.lua:138>
        ...ck/packer/start/telescope.nvim/lua/telescope/command.lua:174: in function 'run_command'
        ...ck/packer/start/telescope.nvim/lua/telescope/command.lua:244: in function 'load_command'
        [string ":lua"]:1: in main chunk
```

running `:Telescope neoclip` seems to work .. so it seems that I get this error because there is no lines yanked in 'unnamed'

Also :Telescope neoclip only works after the :Telescope neoclip unnamed failed

**Comment by desdic** [comment]
Also `:Telescope neoclip` only works after the `:Telescope neoclip unnamed` failed

**Comment by AckslD** [comment]
Thanks for reporting this @desdic, this was indeed a bug and should be fixed by #49.

I'm working on adding more tests to this plugin which will hopefully makes this less likely to happen in the future :)
